kube-client/src/config/file_config.rs

```diff
@@ -688,8 +708,8 @@ users:
         let cfg = Kubeconfig::from_yaml(config_yaml)?;
 
         // Ensure we have data from both documents:
-        assert_eq!(cfg.clusters[0].name, "k3d-promstack");
-        assert_eq!(cfg.clusters[1].name, "k3d-k3s-default");
+        assert_eq!(cfg.clusters[0].name.as_ref().unwrap(), "k3d-promstack");
+        assert_eq!(cfg.clusters[1].name.as_ref().unwrap(), "k3d-k3s-default");
```

Comment:
this stuff expands into something pretty big after running `just fmt`, maybe factor out `let cluster1 = config.clusters[1].cluster.as_ref().unwrap();` (and run `just fmt` having nightly rust available to fix ci)

Reply:
Fixed it, but fmt didn't touch that line, it was on 657, I guess that's the one you were referring to?